테스트 코드 작성

woomae opened the pull request:
- 전남 시군 모든지역에 대해 필터링 테스트

woomae committed:
테스트 코드 작성

- 전남 시군 모든지역을 필터링 테스트

lightpurple commented on test/integration/stores.test.ts:
```diff
@@ -37,96 +37,145 @@ describe(`/v1/stores API Test`, async() => {
+                "진도",
+                "신안",
+            ];
+            for (const city of cities) {
```
for문을 통해 모든 city에 대해 request를 보내는 것보단 faker.helpers.arrayElement 를 사용해서 랜덤한 city에 대해 확인하는 것도 좋아보입니다.

woomae replied:
알겠습니다!

lightpurple commented on test/integration/stores.test.ts:
```diff
@@ -37,96 +37,145 @@ describe(`/v1/stores API Test`, async() => {
+                    .get("/v1/stores")
+                    .set("Accept", "application/json")
+                    .query({
+                        q: storeName,
```
지금 해당 city에 storeName을 가진 가맹점을 조회하는 거 같은데 지금 테스트 목적이 모든 city에 해당 storeName을 가진 가맹점이 있는 걸 테스트하는게 아니라 request에 city 변수를 입력했을때 해당 city의 가맹점들만 응답하는지 테스트를 하는 거기 때문에 테스트 목적에 맞게끔 수정이 필요해 보입니다.

woomae replied:
제가 잘못 이해하고 있었네용...ㅎㅎ
덕분에 q파라미터 없이 검색하는 로직 부분에서 문제를 찾았습니다 수정하겠습니다!!!

woomae committed:
테스트 코드 수정

- city파라미터만을 이용해 값을 검색하는 로직으로 변경
- city파라미터만으로도 findAll함수를 사용하기위해 findAllCity함수 추가
- 기존 함수명중 searchcity를 searchCity로 리네임
- 테스트 케이스가 너무많아 랜덤한 5개의 시군만 체크하도록 변경
